Content/Items/Misc/Accessories.PhilosophersMortar.cs

```diff
@@ -0,0 +1,75 @@
+﻿using NetEasy;
+using StarlightRiver.Content.Items.BaseTypes;
+using StarlightRiver.Content.WorldGeneration;
+using StarlightRiver.Core;
+using StarlightRiver.NPCs;
+using System;
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+using Terraria.GameContent.ItemDropRules;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace StarlightRiver.Content.Items.Misc
+{
+    public class PhilosophersMortar :  CursedAccessory
+    {
+        public override string Texture => AssetDirectory.MiscItem + Name;
+
+        public PhilosophersMortar() : base(ModContent.Request<Texture2D>(AssetDirectory.MiscItem + "PhilosophersMortar").Value) { }
+
+        public override void SetStaticDefaults()
+        {
+            DisplayName.SetDefault("Philosopher's Mortar");
+            Tooltip.SetDefault("Life hearts decrease duration of Potion Sickness by 10 seconds, but heal half as much");
+        }
+
+        public override void SafeSetDefaults()
+        {
+            Item.value = 1;
+            Item.rare = ItemRarityID.LightRed;
+        }
+
+        public override void SafeUpdateEquip(Player Player)
+        {
+            Player.GetModPlayer<PhilMortarPlayer>().equipped = true;
+        }
+    }
+
+    public class PhilMortarPlayer : ModPlayer
+    {
+        public bool equipped = false;
+
+        public override void ResetEffects()
+        {
+            equipped = false;
+        }
+    }
+
+    public class PhilMortarGItem : GlobalItem
```

Comment:
This is an incorrect design pattern. You should use or add to the StarlightPlayer hooks.